packages/terra-form-select/src/search/Frame.jsx

```diff
@@ -410,9 +417,14 @@ class Frame extends React.Component {
       isFocused: document.activeElement === this.input || document.activeElement === this.select,
       isOpen: false,
       isPositioned: false,
-      hasSearchChanged: false,
-      searchValue: '',
     });
+    if (!this.hasEscPressed) {
+      this.setState({
+        hasSearchChanged: false,
+        searchValue: '',
+      });
+      this.props.onSelect('');
```

Comment:
```suggestion
      this.props.onSelect('');
```
why  `onSelect` when we are clearing the value

Reply:
This change is added to clear the selected value from the dropdown when we press the esc key. This onSelect handler clears the state binded to input field. Thanks

Reply:
OnSelect()  is callback provided from consuming application I don't think we need to trigger this for clearing the value

Reply:
Updated. Thanks
cb3824e9ed2ce3f1ed0e175cd02c7d61318c7629